package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lob/ui-components",
-  "version": "0.0.189",
```

Comment:
You should also be updating the version number in `package-lock.json` so they don't get out of sync with each other.